lib/puppetserver/ca/local_certificate_authority.rb

```diff
@@ -39,6 +39,10 @@ def initialize(digest, settings)
         @settings = settings
       end
 
+      def errors
+        @host.errors
```

Comment:
This seems good. It'd be easy to add any additional errors to the ones from host if this class ever needed to return any of its own.